python/lsst/sims/catalogs/generation/db/ObservationMetaData.py

```diff
@@ -60,70 +60,94 @@ class ObservationMetaData(object):
 
     """
     def __init__(self, boundType=None, boundLength=None,
-                 mjd=None, unrefractedRA=None, unrefractedDec=None, rotSkyPos=0.0,
-                 bandpassName='r', phoSimMetadata=None, site=None, m5=None, skyBrightness=None):
+                 mjd=None, unrefractedRA=None, unrefractedDec=None, rotSkyPos=None,
+                 bandpassName=None, phoSimMetaData=None, site=Site(), m5=None, skyBrightness=None):
 
-        self.bounds = None
-        self.boundType = boundType
-        self.mjd = mjd
-        self.bandpass = bandpassName
-        self.skyBrightness = skyBrightness
+        self._bounds = None
+        self._boundType = boundType
+        self._mjd = mjd
+        self._bandpass = bandpassName
+        self._skyBrightness = skyBrightness
+        self._site = site
 
         if rotSkyPos is not None:
-            self.rotSkyPos = numpy.radians(rotSkyPos)
+            self._rotSkyPos = numpy.radians(rotSkyPos)
         else:
-            self.rotSkyPos = None
+            self._rotSkyPos = None
 
         if unrefractedRA is not None:
-            self.unrefractedRA = numpy.radians(unrefractedRA)
+            self._unrefractedRA = numpy.radians(unrefractedRA)
         else:
-            self.unrefractedRA = None
+            self._unrefractedRA = None
 
         if unrefractedDec is not None:
-            self.unrefractedDec = numpy.radians(unrefractedDec)
+            self._unrefractedDec = numpy.radians(unrefractedDec)
         else:
-            self.unrefractedDec = None
+            self._unrefractedDec = None
```

Comment:
I am suggesting
```
def __init__(... ):
      if rotSkyPos is not None:
             self._rotSkyPos = numpy.radians(rotSkyPos):
      else:
-            self._rotSkyPos = rotSkyPos

    @property
    def rotSkyPos(self):
           return self._rotSkyPos
```
This saves one from setting both self._rotSkyPos and self.rotSkyPos

Also, we seem to be using numpy.degrees /numpy.radians frequently but have to wrap an if statement due to the None type (I know I have done this, and you are having to do this again ... should we consolidate by adding a function to sims_utils that operates on None types as well?